src/index.html

```diff
@@ -17,6 +17,43 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1></h1>
```

Comment:
The `h1` tag is empty. Make sure to provide a title or remove the tag if it's not needed.